src/mpyl/cli/commands/health/checks.py

```diff
@@ -53,24 +54,32 @@ def perform_health_checks(bare_console: Console, is_ci: bool = False):
     __check_version(console)
 
     console.title("MPyL configuration")
-    __check_config(
+    if config_path := __validate_config_path(
```

Comment:
Huh I didn't know we have the walrus operator in python

Reply:
Yes, I like it, reminds me of pascal 😄 . And (way too) hard fought https://medium.com/pythonistas/the-story-of-guido-van-rossum-and-his-walrus-operator-quick-blog-a0481af61bd8